libs/candles-chart/src/lib/candles-chart.tsx

```diff
@@ -168,6 +167,11 @@ export const CandlesChartContainer = ({
             chartType: chartType,
             overlays: overlays,
             studies: studies,
+            notEnoughDataText: (
+              <span className="text-xs text-center text-gray-800 dark:text-gray-200">
```

Comment:
chore(blocking): Could you switch to using text-neutral, we use those everywhere else